### PR TITLE
docs: add a blog post for the new feature SSO

### PR DESCRIPTION
**Pull request by Candybase**

### CODE REVIEW

File: content/blog/introducing-single-sign-on-in-bytebase.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Introducing Single Sign-On in Bytebase
+author: Candy
+published_at: 2023/03/07 17:37:17
+feature_image: _static_blog_introducing-single-sign-on-in-bytebase_cover.webp
```
Comment (adela-bytebase) [inline content/blog/introducing-single-sign-on-in-bytebase.md:5]: the cover image is broken

Comment (Candybase) [reply content/blog/introducing-single-sign-on-in-bytebase.md]: Already fixed it.

File: content/blog/introducing-single-sign-on-in-bytebase.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: Introducing Single Sign-On in Bytebase
+author: Candy
+published_at: 2023/03/07 17:37:17
+feature_image: /static/blog/introducing-single-sign-on-in-bytebase/cover.webp
+tags: Announcement
+featured: true
+description: This article demonstrates the Single Sign-On protocols supported by Bytebase. With SSO, users log in once and access all authorized resources, resulting in improved security, user experience, and organizational flexibility.
```
Comment (tianzhou) [inline content/blog/introducing-single-sign-on-in-bytebase.md:8]:
```suggestion
description: This article demonstrates the Single Sign-On (SSO) protocols supported by Bytebase. SSO allows users to log in once and access all authorized resources, improving security, user experience, and organizational flexibility.
```

File: content/blog/introducing-single-sign-on-in-bytebase.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+## What’s Single Sign-On?
+
+Single Sign-On (SSO) is a user authentication service that allows users to access multiple applications or services with one set of login credentials. There are several benefits to supporting SSO:
```
Comment (tianzhou) [inline content/blog/introducing-single-sign-on-in-bytebase.md:13]:
```suggestion
Single Sign-On (SSO) is a user authentication service allowing users to access multiple applications or services with one login credential. There are several benefits to supporting SSO:
```

File: content/blog/introducing-single-sign-on-in-bytebase.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* Google
+* Custom
+
+For details information on how to configure SSO using OAuth 2.0, you can refer to the user doc [OAuth 2.0](/docs/administration/sso/oauth2).
+
+### OpenID Connect (OIDC)
+
+OIDC is an open standard for authentication and authorization that is built on top of OAuth 2.0. It provides a secure and reliable way for users to authenticate to an application using their existing online accounts.
+
+Like OAuth 2.0, OIDC relies on identity providers to authenticate users. These identity providers can be third-party services or self-hosted solutions. Byteabase provides the following identity providers for OIDC:
```
Comment (tianzhou) [inline content/blog/introducing-single-sign-on-in-bytebase.md:40]:
```suggestion
Like OAuth 2.0, OIDC relies on identity providers to authenticate users. These identity providers can be third-party services or self-hosted solutions including:
```

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/blog/introducing-single-sign-on-in-bytebase.md (hunk 4 above)
Comment (Candybase) [reply content/blog/introducing-single-sign-on-in-bytebase.md:40]: Bytebase just provides part of identity providers. So I prefer the original expression.

**Pull request merged**